skyportal/handlers/api/filter.py

```diff
@@ -139,7 +141,7 @@ def post(self):
             return self.error("Access to stream unauthorized for group.")
 
         DBSession().add(fil)
-        DBSession().commit()
+        self.finalize_transaction()
```

Comment:
Just a note (mostly for myself) that we're replacing this file on F, so relevant changes need to made there. I'm actually working on a PR that fiddles with this, so we'll take care of it.